firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -475,6 +478,18 @@ static void publish_ps_ctrl_brake_report( void )
     // Set override flag
     data->override = local_override;
 
+    // Set enabled flag
+    data->enabled = (uint8_t) controlEnabled;
+
+    // Set pedal input
+    data->pedal_input = brake_pressure_can;
```

Comment:
Should I modify this PR to include a struct to house this data? How should I resolve this?